### PR TITLE
PR for Home page (Refactored Home page)

### PR DESCRIPTION
**Pull request by kiranpratyush**
Refactored the Home page and added some simple responsiveness. Please review it.

[Home page](https://testing2322.netlify.app/)

image
image
image

### CODE REVIEW

File: general.css [hunk 1]
```diff
@@ -130,3 +129,29 @@ footer {
...
+  }
+  main{
+    padding:15px;
+  }
+  .footer-container{
+    justify-content:flex-start;
+    gap:20px;
+  }
+  .header-top {
+    padding: 15px;
```
Comment (Varu98) [inline general.css:145]: Good work on media queries 👍🏻

**Review by Varu98 — APPROVED** [review]
I am curious about the use of px in the files instead of rems. would love to get an insight 🤓

**Review by someshtiwarii — APPROVED** [review]

File: general.css [hunk 2]
```diff
@@ -130,3 +129,29 @@ footer {
...
+
+@media screen and (max-width: 550px) {
+  body{
+    font-size: 1.2rem;
+  }
+  main{
+    padding:15px;
+  }
+  .footer-container{
+    justify-content:flex-start;
```
Comment (MarkVed17) [inline general.css:141]: Apart from the code formatting here in the CSS file, everything else looks good. The UI looks very nice @kiranpratyush 👍🏼

File: general.css (hunk 1 above)
Comment (kiranpratyush) [reply general.css:145]: Sure will discuss

**Pull request merged**